hack/charts/cluster-api-operator/values.yaml

```diff
@@ -23,8 +23,9 @@ image:
     pullPolicy: IfNotPresent
 env:
   manager: []
-healthAddr: ":8081"
 diagnosticsAddress: ":8443"
+healthAddr: ":9440"
+metricsBindAddr: "127.0.0.1:8080"
```

Comment:
why are we adding this back, where we just removed it in https://github.com/kubernetes-sigs/cluster-api-operator/pull/665/files#diff-a9d2a05c9ca5b0702274afd204a118bf07760380a9809fac8a534be41bc14489L27?

Reply:
It seems to be a bad rebase, fixed.